packages/components/DigitBox.tsx

```diff
@@ -0,0 +1,20 @@
+import { Flex, Heading, Input, Spacer } from '@chakra-ui/react';
+
+export default function DigitBox() {
+  return (
+    <Flex
+      w="27vw"
+      h="35vh"
+      justifyContent="center"
+      alignItems="center"
+      px="20px"
+    >
+      <Heading>Número</Heading>
+      <Flex padding="3rem">
+        <Input width="50px" height="70px" border="1px solid black"></Input>
```

Comment:
se nao me engano o input é self closing
```suggestion
        <Input width="50px" height="70px" border="1px solid black" />
```